services/motion/builtin/move_request.go

```diff
@@ -258,6 +282,8 @@ func (ms *builtIn) newMoveOnGlobeRequest(
 		return nil, err
 	}
 
+	// TODO: I don't fully understand this. What does it mean that heading is not taken into account? What does transformed mean?
+	// How do you transform something based on something else?
```

Comment:
"heading is not taken into account" means that since heading information is not part of a `geo.Point`, the Pose returned is merely the XY displacement between the points and cannot be used to say "a robot at this geopoint will go to that pose and thus be at that geopoint" if the robot uses our convention of Y=forwards; it must be adjusted after the fact to account for which way the robot is facing.

Reply:
Got it. Isn't it somewhat misleading for `GeoPointToPose` to return a pose then if it has no orientation information?

Reply:
I would agree with that, yes. It would make more sense for it to return an `r3.Vector`.

Though that vector does need to be converted to a `Pose` in order to be made useful.